python/tests/test_account.py

```diff
@@ -893,6 +893,33 @@ def test_send_self_message(self, acfactory, lp):
         chat.send_text("hello")
         ac1._evtracker.get_matching("DC_EVENT_SMTP_MESSAGE_SENT")
 
+    def test_decrypt_self_message(self, acfactory, lp):
+        ac1 = acfactory.get_online_configuring_account()
+        # We are actually not using ac2, we just need an address to send a message to
+        ac2 = acfactory.get_online_configuring_account()
+        ac3 = acfactory.clone_online_account(ac1)
```

Comment:
This is an expected behavior of `clone_online_account`. It only copies address and password, but not the keys. Optional argument `pre_generated_key` controls whether pre-generated key is used or not, so it can't help either. In this case ac1 gets alice key, ac2 gets bob key and ac3 gets charlie key.

If you want ac3 to be able to read messages, you need to either send an autocrypt setup message or simulate backup export and import.

Reply:
Thanks for noting ... i think clone should actually clone the key, or at least have a flag for doing that.

Reply:
It should be renamed and documented instead. It is already used in several places to emulate fresh reinstall with user typing in the password but not importing the key.

Reply:
Another helper method for reinstallation via backup could be useful, but it is a completely different flow.

Reply:
Ah ok, sorry for the confusion.

So, I'll add a new PR that modifies the docs and close this one for now; maybe I will make similar tests one day that use imex.